ananse/influence.py

```diff
@@ -456,6 +457,120 @@ def save_reg_network(self, filename, full_output=False):
                         d["TF_act_target"],
                     ]
                     nw.write("\t".join(str(v) for v in cols) + "\n")
+#def plot_TF_GRN(self, infile, outfile, GRN_wb = False, network_algorithm = 'neato', n_TFs = 20, cmap = 'viridis', full_output = False):
+    def plot_TF_GRN(self, infile, outfile, GRN_wb = False, network_algorithm = 'neato', n_TFs = 20, cmap = 'viridis', full_output = False, cutoff_val = 0.1):
+        """
+        Plot the top20 of differential TFs and their interactions with the highest interaction score into a GRN network image
+
+        Parameters
+        ----------
+        self: influence_object to extract the diff network self.G, #future optimization = read diffnetwork instead
+        infile: influence output file
+        outfile: output file location
+        GRN_wb: when true and a -full--output influence GRN is suplied, use weighted binding interaction score to vizualize
```

Comment:
I think this argument is unclear, and would personally go for `edge_score = "weigth"` and explain in the docstrings options are `"weight" (probability), "wb" (weighted binding)`.

This is a subjective remark. Feel free to ignore if it makes no sense/is too much work.

Reply:
I'll change it :)